Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
-name = "config"
-version = "0.11.0"
+authors = ["Ryan Leckey <leckey.ryan@gmail.com>"]
+categories = ["config"]
 description = "Layered configuration system for Rust applications."
+edition = "2018"
 homepage = "https://github.com/mehcode/config-rs"
-repository = "https://github.com/mehcode/config-rs"
-readme = "README.md"
 keywords = ["config", "configuration", "settings", "env", "environment"]
-authors = ["Ryan Leckey <leckey.ryan@gmail.com>"]
-categories = ["config"]
 license = "MIT/Apache-2.0"
-edition = "2018"
+name = "config"
+readme = "README.md"
+repository = "https://github.com/mehcode/config-rs"
+version = "0.11.0"
 
 [badges]
-maintenance = { status = "actively-developed" }
+maintenance = {status = "actively-developed"}
```

Comment:
All of the changes above are not necessary for the PR, please revert them back!

Reply:
Done